plugin_manager.py

```diff
@@ -89,8 +136,9 @@ def start(self):
                 # Load the next plugin and give it the current panel layout
                 current_module = next_module
                 current_plugin = imp.load_source("__init__", next_module["path"]).plugin()
```

Comment:
I forgot to mention that `imp.load_source` is deprecated. See #20 for how I solved the deprecation issue.